integration/integration_test.go

```diff
@@ -238,35 +240,22 @@ func (s *BaseSuite) startWhoami(c *check.C) {
 	c.Assert(err, checker.IsNil)
 }
 
-func (s *BaseSuite) installTiller(c *check.C) {
-	// create tiller service account.
-	cmd := exec.Command("kubectl", "apply",
-		"-f", path.Join(s.dir, "resources/helm/serviceaccount.yaml"))
+func (s *BaseSuite) createMaeshNamespace(c *check.C) {
+	// Create maesh namespace, required by helm v3.
+	cmd := exec.Command("kubectl", "create", "namespace", maeshNamespace)
```

Comment:
Why not use the kube client library for the integration tests instead of `kubectl`? (I know it wasnt introduced in this PR but still, I'd like to know :p)

Reply:
Very good question.

I will open a new issue to update and consolidate the commands to use the client lib

Reply:
#382